libtsuba/src/AddProperties.cpp

```diff
@@ -130,44 +126,16 @@ katana::AddProperties(
               return KATANA_CHECKED_CONTEXT(
                   LoadProperties(prop->name(), path), "error loading {}", path);
             });
-    auto on_complete = [add_fn, prop, cache,
-                        rdg](const std::shared_ptr<arrow::Table>& props)
+    auto on_complete = [add_fn, is_property,
+                        prop](const std::shared_ptr<arrow::Table>& props)
         -> katana::CopyableResult<void> {
-      if (cache != nullptr) {
-        auto& tracer = katana::GetTracer();
-        // Do not put uint8 types in property cache.  Users cannot create uint8
-        // properties via Cypher, but they can create them via parquet import.
-        if (props->column(0)->type()->Equals(arrow::uint8())) {
-          KATANA_WARN_ONCE(
-              "deprecated graph format; type is uint8: {}",
-              props->field(0)->name());
-          KATANA_LOG_VASSERT(
-              !rdg->IsEntityTypeIDsOutsideProperties(),
-              "storage_format_version >= 2 RDG may not have uint8 type "
-              "properties");
-        } else {
-          // Only match properties from the same RDG prefix
-          katana::Uri cache_key(rdg->rdg_dir().Join(prop->path()));
-          cache->Insert(cache_key, props);
-          tracer.GetActiveSpan().Log(
-              "addproperties property cache insert",
-              {
-                  {"name", prop->name()},
-                  {"approx_size", katana::ApproxTableMemUse(props)},
-                  {"approx_size_human",
-                   katana::BytesToStr(
-                       "{:.2f}{}", katana::ApproxTableMemUse(props))},
-              });
-        }
-      }
       KATANA_CHECKED_CONTEXT(
           add_fn(props), "adding {}", std::quoted(prop->name()));
       prop->WasLoaded(props->field(0)->type());
-      if (prop->type()->Equals(arrow::uint8())) {
```

Comment:
Don't we still need checks to ensure we don't add uint8 properties, or do we enforce this elsewhere now?